errors/errors.go

```diff
@@ -53,4 +53,5 @@ var (
 	ErrRegistryNoContent       = errors.New("sync: could not find a Content that matches localRepo")
 	ErrSyncSignatureNotFound   = errors.New("sync: couldn't find any upstream notary/cosign signatures")
 	ErrSyncSignature           = errors.New("sync: couldn't get upstream notary/cosign signatures")
+	ErrImageLintAnnotations    = errors.New("routes: missing mandatory annotations")
```

Comment:
This error with this message should not be defined here. It's specific to one particular linter which we optionally include in the build.
We could have a more generic message here.